.gitignore

```diff
@@ -1,2 +1,3 @@
 *.rdb
 testdata/*/
+.idea/
```

Comment:
Please revert - we don't add every editor to gitignore file. The better alternative is to use global gitignore - https://help.github.com/articles/ignoring-files/#create-a-global-gitignore

Reply:
No problem, reverted